src/common.rs

```diff
@@ -1,7 +1,59 @@
 //! Common types and definitions.
 
+use bitflags::bitflags;
+
 /// Thread ID
 pub type Tid = core::num::NonZeroUsize;
 
 /// Process ID
 pub type Pid = core::num::NonZeroUsize;
+
+bitflags! {
```

Comment:
please move the defns for `HostOpenFlags` and `HostMode` to `target::ext::host_io::`